### DISCUSSION TITLE
Questão TI21#1

### DISCUSSION THREAD

**Discussion by paulocafee** [paulowh/TI21#7] [Q&A] [answered]
Quais são as principais diferenças entre o método GET e POST em HTML?

**Comment by Arrowaaa** [comment]
O Método GET é utilizado para obter dados, enquanto o método POST é utilizado para enviar dados.

**Comment by Guilhlm** [comment]
get obtem os dados e post realiza o seu envio

**Comment by riichardgabriel** [comment, marked as answer]
GET – é usado quando o cliente deseja obter recursos do servidor.
POST – é usado quando o cliente deseja enviar dados para processamento ao servidor, como os dados de um formulário, por exemplo.
